res/css/views/voice_messages/_PlayPauseButton.scss

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_PlayPauseButton {
+    position: relative;
+    width: 32px;
+    height: 32px;
+    border-radius: 32px;
```

Comment:
To avoid this value going out of sync you can use `50%` here and it will always form a circle when the width and height are matching

Reply:
There's a lot of things we're supposed to do regarding border radiuses, but this is the consistent approach for us. I'd like to see us fix it everywhere if we're going to fix it

Reply:
more accurately, I'd like to see us decide and codify the approach for this. I'm happy to fix this up (and the translate3d below) in a later PR if we deem this Not The Way™